learn/validator-trustscore.md

```diff
@@ -66,66 +56,77 @@ $$
 
 #### Dominance Ratio
 
-To find the dominance ratio ($s$) of a validator, we have two methods:
+The dominance ratio ($s$) of a validator is calculated using one of two methods, depending on the epoch's state. However, due to technical limitations, only the **Active Epoch Method** is currently implemented and used.
 
-1. **First method**: Calculate the ratio by dividing the validator's share by the total share of the network for an active epoch:
+**Active Epoch Method**:
+During an active epoch, the dominance ratio is determined by dividing the validator's share by the total network share:
 
 $$
 s = \frac{v}{Z}
 $$
 
-Where $v$ is the validator's share and $Z$ is the total network share. This method applies when the epoch is active, so we can access each validator's balance using the `getActiveValidators` function from the RPC.
+Where:
+- $v$: The validator's share.
+- $Z$: The total network share.
 
-2. **Second method**: A different approach is used for a closed epoch. This is less accurate due to some randomness and is considered a fallback option. Here we look at the slot distribution of each voting block, which reflects the amount staked by each validator. The dominance ratio is calculated by dividing the number of slots allocated to a validator by the total number of slots in that epoch:
+This method relies on the `getActiveValidators` function from the [RPC](https://www.nimiq.com/developers/build/set-up-your-own-node/rpc-docs/#getactivevalidators), which provides real-time balances of each active validator.
+
+**Finished Epoch Method**:
+For a closed epoch, a less precise fallback method exists, but is not currently implemented. This approach calculates the dominance ratio based on slot distribution across voting blocks. The ratio is derived by dividing the number of slots allocated to a validator by the total slots in the epoch:
 
 $$
 s = \frac{sl}{Sl}
 $$
 
-Where $sl$ is the number of slots allocated to the validator and $Sl$ is the total number of slots.
+Where:
+
+- $sl$: The number of slots allocated to the validator.
+- $Sl$: The total number of slots in the epoch.
 
-> The second method in the code is called `dominanceRatioViaSlots`.
+While this method is defined in the code as `dominanceRatioViaSlots`, it is not actively used in practice.
 
 #### Curve adjustment
 
-Then, we apply a curve to the stake percentage to calculate the dominance score ($S$):
+After determining the dominance ratio, a curve is applied to calculate the dominance score $S$:
 
 $$
 S = \max \left( 0 , 1 - s^{k} \right), \quad \text{being } t = 0.15 \text{ and } k=7.5
 $$
 
-Where $t$ is the threshold and $k$ is the slope of the curve.
+Where:
+
+- $t = 0.25$ is the threshold.
```

Comment:
```suggestion
- $t = 0.15$ is the threshold.
```

Reply:
![image](https://github.com/user-attachments/assets/78fb9162-2450-4233-8833-d1c23c05b3f1)
@onmax send me these updated values in the Hackathon
(?)

Reply:
This is the one: https://www.desmos.com/calculator/xynm6uphlq